expr_test.go

```diff
@@ -1273,6 +1273,18 @@ func TestExpr(t *testing.T) {
 			`1 < 2 < 3 == true`,
 			true,
 		},
+		{
+			`[1, 2, 3]?.[5]`,
+			nil,
+		},
+		{
+			`'string'?.[5]`,
+			"g",
+		},
+		{
+			`'string'?.[7]`,
```

Comment:
This is not exactly correct behavior I'm looking for. 

In case `array?.[7]` operator `?.` checks what array itself defined and not nil.

So the code

```
array?.[7]
```
will panic in there is no 7 element index. 

What we can do is to alter `[7]` behavior with operator `?.` **after** it.
 
```
array[7]?.field
```

In this case we can alter OpFetch and do not panic if 7 index does not exist. 

This mean we, unfortunately, does not have a syntax to alter `[7]` behavior if it is the last postfix op.

But we can alter the whole "chain" in presence of `??` operator! 

```js
array[7].field ?? 'default'
```